.mergify.yml

```diff
@@ -1,14 +1,18 @@
+queue_rules:
+  - name: default
+    conditions:
+      - status-success=continuous-integration/travis-ci/pr
```

Comment:
you don't want to queue before CI passes so you should keep that in the conditions for being in queue